packages/sfcc/src/api/endpoints/catalog/products/get-products.ts

```diff
@@ -0,0 +1,46 @@
+import { normalizeSearchProducts } from '../../../utils/normalise-product';
+import { ProductsEndpoint } from '.'
+
+
+const getProducts: ProductsEndpoint['handlers']['getProducts'] = async ({
+  req,
+  res,
+  body: { search, categoryId, brandId, sort },
+  config
+}) => {
+  const { sdk  } = config;
+
+  let searchTerm = search;
+
+  if (categoryId) {
+    searchTerm = categoryId as string
+  }
+
+  // set default for main category
+  if (!searchTerm) {
+    searchTerm = 'clothing'
+  }
+
+  const searchClient = await sdk.getSearchClient();
+  // use SDK search API for initial products
+  const searchResults = await searchClient.productSearch({
+      parameters: {
+          q: searchTerm,
+          limit: 20
+      }
+  });
+  let products = [];
+  let found = false;
+  if (searchResults.total) {
+      found = true;
+      products = normalizeSearchProducts(searchResults.hits) as any[];
+  } else {
+      // TODO: handle this better?
+      console.log("No results for search");
+  }
```

Comment:
Can this check only happen if there's an error?

Reply:
Actually, it's not an error so nothing to handle here. The `found` is set to false from the start so this `else` block is not needed at all.